main/test/flix/Test.Assoc.Type.flix

```diff
@@ -16,4 +16,20 @@ mod Test.Assoc.Type {
         pub def head(x: String): Option[Char] = if (String.isEmpty(x)) None else Some(String.charAt(0, x))
     }
 
+    class Collection2[t] {
+        type Elm
```

Comment:
Maybe we can have an intermediate test where the kind is present? :) So we have 3 tests.